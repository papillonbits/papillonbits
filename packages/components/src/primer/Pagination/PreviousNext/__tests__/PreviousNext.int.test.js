import { regular, inactive } from './PreviousNext.int.story'

describe('<PreviousNext />', () => {
  afterEach(() => jest.clearAllMocks())

  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })

    test('must match inactive()', () => {
      expect(global.renderToJSON(inactive())).toMatchSnapshot()
    })
  })
})
