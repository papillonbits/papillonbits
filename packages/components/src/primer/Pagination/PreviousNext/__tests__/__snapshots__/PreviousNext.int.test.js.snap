// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PreviousNext /> Render must match inactive() 1`] = `
<div>
  <nav
    aria-label="Pagination"
    class="paginate-container previousnext-inactive"
  >
    <div
      class="pagination d-flex"
    >
      <a
        aria-label="Previous"
        href="#url"
        rel="previous"
      >
        &lt;
      </a>
      <span>
        Viewing 3 to 7
      </span>
      <a
        aria-label="Next"
        href="#url"
        rel="next"
      >
        &gt;
      </a>
    </div>
  </nav>
</div>
`;

exports[`<PreviousNext /> Render must match regular() 1`] = `
<div>
  <nav
    aria-label="Pagination"
    class="paginate-container"
  >
    <div
      class="pagination d-flex"
    >
      <a
        aria-label="Previous"
        href="#url"
        rel="previous"
      >
        &lt;
      </a>
      <span>
        Viewing 3 to 7
      </span>
      <a
        aria-label="Next"
        href="#url"
        rel="next"
      >
        &gt;
      </a>
    </div>
  </nav>
</div>
`;
